content/swagger/resources/measurement_families/definitions/measurement_family.yaml

```diff
@@ -0,0 +1,81 @@
+type: object
```

Comment:
I don't think we'll have a unitary measurement family endpoint :/. Yet I don't know how swagger specification works :)

Reply:
It's only a list